src/renderer/components/watch-video-comments/watch-video-comments.css

```diff
@@ -70,8 +70,7 @@
   font-weight: bold;
   font-size: 14px;
   margin-inline-start: 68px;
-  margin-block-start: 0px;
-  overflow: hidden;
+  margin-block-start: 0;
   text-overflow: ellipsis;
```

Comment:
```suggestion
  word-wrap: break-word;
```

Reply:
Got example for long author name?
Neither works for my fake long name but I have no idea what real long name looks like
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/059f875c-acdc-4c7c-a180-8e8804cd612a)

Reply:
Try testing at non-full-device width; don't know of any examples in YouTube at the moment